web/templates/settings/notification_settings.hbs

```diff
@@ -29,7 +29,13 @@
             <tbody>
                 {{#each general_settings}}
                     <tr>
-                        <td>{{ this.label }}</td>
+                        <td>
+
```

Comment:
Is this a stray blank line?

Reply:
The blank line was added by the linter during auto-formatting.